src/graphql/SchemaGql.js

```diff
@@ -0,0 +1,50 @@
+import gql from "graphql-tag";
+
```

Comment:
I think that all of the queries here are no longer used buy I still kept them here in case will need them